integration/build/build_test.go

```diff
@@ -166,16 +169,27 @@ func TestBuildMultiStageParentConfig(t *testing.T) {
 	resp.Body.Close()
 	assert.NilError(t, err)
 
+	time.Sleep(30 * time.Second)
```

Comment:
Was this sleep needed on Linux as well?

Reply:
It was added by either Vincent or JHoward so I'm really not certain. Things work with it and presumably didn't work without it, but I'm not certain of the origin. It can be re-evaluated later it need be.